lib/qx/tool/cli/commands/Serve.js

```diff
@@ -151,11 +150,11 @@ qx.Class.define("qx.tool.cli.commands.Serve", {
         app.use('/', express.static(target.getOutputDir()));
       } else {
         app.use('/', express.static(path.join(__dirname, "../../../../../tool/cli/serve/build")))
-        app.use('/' + target.getOutputDir(), express.static(target.getOutputDir()));
+        app.use('/build/', express.static(target.getOutputDir()));
```

Comment:
isn't this hard coding the target directory?

Reply:
No only the url path to the server, All apps then are reached by http://localhost:8080/build/app.
If you use  target-getOutputDir() here with a traget path like '../../build' the serve will not build.
I think a hard path on the webserver is no problem.